engine/apps/mobile_app/urls.py

```diff
@@ -1,13 +1,15 @@
-from apps.mobile_app.views import APNSDeviceAuthorizedViewSet, GCMDeviceAuthorizedViewSet, MobileAppAuthTokenAPIView
+from apps.mobile_app.fcm_relay import FCMRelayView
+from apps.mobile_app.views import APNSDeviceAuthorizedViewSet, FCMDeviceAuthorizedViewSet, MobileAppAuthTokenAPIView
 from common.api_helpers.optional_slash_router import OptionalSlashRouter, optional_slash_path
 
 app_name = "mobile_app"
 router = OptionalSlashRouter()
 
 router.register("apns", APNSDeviceAuthorizedViewSet, basename="apns")
-router.register("gcm", GCMDeviceAuthorizedViewSet, basename="gcm")
+router.register("fcm", FCMDeviceAuthorizedViewSet, basename="fcm")
 
 urlpatterns = [
     *router.urls,
     optional_slash_path("auth_token", MobileAppAuthTokenAPIView.as_view(), name="auth_token"),
+    optional_slash_path("fcm_relay", FCMRelayView.as_view(), name="fcm_relay"),
```

Comment:
should this route be disabled for OSS installations to avoid confusion?

Reply:
Good call! Added a new env var `FCM_RELAY_ENABLED` that defaults to False in https://github.com/grafana/oncall/pull/923/commits/25fae4933d7bfdafb78b4f8b279d13c733d7e56b